Application/EdFi.Ods.SandboxAdmin/Extensions/HealthCheckServiceExtensions.cs

```diff
@@ -1,4 +1,4 @@
-﻿// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Why are all these headers getting tweaked?

Reply:
Because the original had a BOM, and `dotnet format` removed it.